src/GitVersionCore.Tests/ExecuteCoreTests.cs

```diff
@@ -19,6 +19,26 @@ public void SetUp()
     }
 
     [Test]
+    public void CacheKeySameAfterReNormalizing()
+    {
+        var versionAndBranchFinder = new ExecuteCore(fileSystem);
+
+        RepositoryScope(versionAndBranchFinder, (fixture, vv) =>
+        {
+            var targetUrl = "https://github.com/GitTools/GitVersion.git";
+            var targetBranch = "refs/head/master";
+            var gitPreparer = new GitPreparer(targetUrl, null, new Authentication(), false, fixture.RepositoryPath);
+            //var cacheKey0 = GitVersionCacheKeyFactory.Create(fileSystem, gitPreparer, null);
```

Comment:
I left it there as a reminder that the cache key changes after Initialize, but ok.